### ISSUE TITLE
Add Bug Report page

### ISSUE THREAD

**Issue by yatri1609** [fossasia/badge-magic-android#75]
**Describe the feature you'd like**
Adding bug report or feature request form to get bug reports or requests

**Additional context**
Dependent on #52 where Navigation Drawer will be added

**Would you like to work on the issue?**
Yes

**Comment by mariobehling** [comment]
Low priority. Please focus on moving forward the app through features first:
https://github.com/fossasia/badge-magic-android/labels/Priority%3A%20High

**Comment by yashk2000** [comment]
@yatri1609 @iamareebjamal this has been done. Could you please close this issue?

**Comment by adityastic** [comment]
@iamareebjamal @mariobehling close this
